forward_engineering/ddlProvider/ddlProvider.js

```diff
@@ -385,6 +385,11 @@ module.exports = (baseProvider, options, app) => {
 
 		createIndex(tableName, index) {
 			const indexName = getIndexName({ index });
+
+			if (!indexName.trim() || !index.indxKey.length) {
```

Comment:
I'm wondering how that `indexName.trim()` should work, when I was able to get the following ddl with the _spaced_ index name:

```
CREATE INDEX "new_schema"." " ON "new_schema"."new_table" ("new_column" ASC);
```

Reply:
Thanks! Changed